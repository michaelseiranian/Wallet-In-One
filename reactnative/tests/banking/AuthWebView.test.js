import React from 'react';
import renderer from 'react-test-renderer';
import AuthWebView from '../../screens/banking/AuthView'

describe('<AuthWebView />', () => {
    it('snapshot test', () => {
        const snapshot = renderer.create(<AuthWebView />);
        expect(snapshot).toMatchSnapshot();         
      });
  });