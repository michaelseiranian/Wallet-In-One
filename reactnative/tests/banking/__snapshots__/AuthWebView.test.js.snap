// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthWebView /> snapshot test 1`] = `
<View
  style={
    {
      "backgroundColor": "white",
      "flex": 1,
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "backgroundColor": "white",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <ActivityIndicator
      color="purple"
      size="large"
    />
  </View>
  <View
    accessibilityRole="button"
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        [
          {},
        ]
      }
    >
      <Text
        style={
          [
            {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Cancel
      </Text>
    </View>
  </View>
  <View
    style={
      [
        {
          "flex": 1,
          "overflow": "hidden",
        },
        undefined,
      ]
    }
  >
    <RNCWebView
      cacheEnabled={true}
      injectedJavaScriptBeforeContentLoadedForMainFrameOnly={true}
      injectedJavaScriptForMainFrameOnly={true}
      javaScriptEnabled={true}
      messagingEnabled={true}
      onContentProcessDidTerminate={[Function]}
      onHttpError={[Function]}
      onLoadingError={[Function]}
      onLoadingFinish={[Function]}
      onLoadingProgress={[Function]}
      onLoadingStart={[Function]}
      onMessage={[Function]}
      onShouldStartLoadWithRequest={[Function]}
      source={
        {
          "uri": undefined,
        }
      }
      style={
        [
          {
            "flex": 1,
            "overflow": "hidden",
          },
          {
            "backgroundColor": "#ffffff",
          },
          undefined,
        ]
      }
      textInteractionEnabled={true}
      useSharedProcessPool={true}
    />
  </View>
</View>
`;
